IFC4x3/PropertyConstants/l/LINEAR_1m2i15jzb7DeT8TSDmT03v.xml

```diff
@@ -1,3 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LINEAR_1m2i15jzb7DeT8TSDmT03v" Name="LINEAR" UniqueId="700ac045-b7d9-4736-8748-75c3707400f9" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LINEAR_1m2i15jzb7DeT8TSDmT03v" Name="LINEAR" UniqueId="700ac045-b7d9-4736-8748-75c3707400f9">
+	<Localization>
+		<DocLocalization Locale="en">
+			<Documentation>Specifies that the transition of the superelevation between the previous superelevation event and this event shall be linearly interpolated.
```

Comment:
Merge Conflict - suggestion in other pull request was to combine both LINEAR constants